test/common.py

```diff
@@ -1,9 +1,19 @@
 """Common methods for tests."""
 
 import asyncio
+from typing import Protocol
 from unittest.mock import AsyncMock
 
 
+class MockCommandProtocol(Protocol):
```

Comment:
I added some type annotations for the new test, to not have missing type information.